test/e2e/generator/generate.go

```diff
@@ -62,6 +62,12 @@ func Generate(r *rand.Rand, opts Options) ([]e2e.Manifest, error) {
 		if err != nil {
 			return nil, err
 		}
+
+		if len(manifest.Nodes) == 1 {
```

Comment:
I'm generally in favor of this change. Is there a way we can do this without filling this function with if statements? Maybe just an 'isExcludedCase' function that we can add combos we'd like to skip to that this function calls?

Reply:
Seems a bit premature at this point given that there's only one statement, and one that is going to get deleted on master post-release (because when the legacy stack disappears so do the hybrid tests.) 

If we find other single-node tests to trim out, then maybe it'll make sense to refactor.